src/main/java/seedu/address/model/particpant/Participant.java

```diff
@@ -0,0 +1,169 @@
+package seedu.address.model.particpant;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+
+import seedu.address.model.person.Address;
+import seedu.address.model.person.Email;
+import seedu.address.model.person.Name;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.Phone;
+import seedu.address.model.tag.Tag;
+
+/**
+ * Represents a Participant in an event
+ * Guarantees: name, phone, email, address, tags, birthDate, notes, nextOfKins are present and not null, field values
+ * are validated, immutable.
+ */
+public class Participant extends Person {
+
+    private final BirthDate birthDate;
+    private final Set<Note> notes = new HashSet<>();
+    private final ArrayList<Person> nextOfKins = new ArrayList<>();
+
+
+    /**
+     * Every field must be present and not null.
+     *
+     * @param name       Name object of the person
+     * @param phone      Phone object of the person
+     * @param email      Email object of the person
+     * @param address    Address object of the person
+     * @param tags       tags of the person
+     * @param birthDate  birthdate of the person
+     * @param notes      notes attached by the manager
+     * @param nextOfKins nextOfKins of the person
+     */
+    public Participant(Name name, Phone phone, Email email, Address address, Set<Tag> tags, BirthDate birthDate,
+                       Set<Note> notes, ArrayList<Person> nextOfKins) {
+        super(name, phone, email, address, tags);
+        requireAllNonNull(birthDate, notes, nextOfKins);
+        this.birthDate = birthDate;
+        this.notes.addAll(notes);
+        this.nextOfKins.addAll(nextOfKins);
+    }
+
+
+    /**
+     * @return this object's birthDate
+     */
+    public BirthDate getBirthDate() {
+        return this.birthDate;
+    }
+
+    /**
+     * @return this object's nextOfKins
+     */
+    public ArrayList<Person> getNextOfKins() {
+        return this.nextOfKins;
+    }
+
+
+    /**
+     * Add note to set of notes
+     *
+     * @param note note to be added
+     */
+    public void addNote(Note note) {
+        this.notes.add(note);
+    }
+
+    /**
+     * Remove note from set
+     *
+     * @param note note to be removed
+     */
+    public void removeNote(Note note) {
+        this.notes.remove(note);
+    }
+
+    /**
+     * Returns an immutable note set, which throws {@code UnsupportedOperationException}
+     * if modification is attempted.
+     */
+    public Set<Note> getNotes() {
+        return Collections.unmodifiableSet(notes);
+    }
+
+    /**
+     * Returns true if both participants have the same name.
+     * This defines a weaker notion of equality between two persons.
+     * This allow Participant to both pass in Person and Participant objects.
```

Comment:
oh since Participant extends Person alr so it will also inherit isSamePerson(Person otherPerson) so you can use that to compare the Participant to another Person but yes you are right I should rename this to isSameParticipant and maybe add in a slightly stronger comparison thanks!

Reply:
Oh, my bad, I havent get started on familiarising with the codebase yet, will get started after tues 🤣